packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -399,6 +403,14 @@ class RenderUiKitView extends RenderBox {
     _lastPointerDownEvent = null;
   }
 
+  @override	
+  void describeSemanticsConfiguration (SemanticsConfiguration config) {	
+    super.describeSemanticsConfiguration(config);	
+    config.isSemanticBoundary = true;	
+    config.platformViewId = _viewController.id;	
```

Comment:
For android, we only add the platform id to the tree if the platform view has actually been created and is ready to go (which may be a few frames after the PlatformView widget has been created). I would assume we want a similar behavior on iOS? (unless on iOS view and widget creation happen in the same frame?)

Reply:
I think in iOS, the view controller is only initialized after the view is created. https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/services/platform_views.dart#L133 So it should be safe to assign the platformViewId to _viewController.id directly assuming the view has been successfully created. @amirh could you confirm?